src/employer_identification_number_formatter.js

```diff
@@ -0,0 +1,43 @@
+import DelimitedTextFormatter from './delimited_text_formatter';
+
+/**
+ * @const
+ * @private
+ */
+var DIGITS_PATTERN = /^\d*$/;
```

Comment:
Prefer `const` over var here.

PS I know this is from SSN but we still have some cleanup to do there, I'd rather add new stuff in the best ES6 style possible.